python_modules/dagster/dagster/grpc/types.py

```diff
@@ -297,6 +297,14 @@ def __new__(
         )
 
 
+@whitelist_for_serdes
+class NotebookDataArgs(namedtuple("_NotebookDataArgs", "notebook_path")):
+    def __new__(cls, notebook_path):
+        return super(NotebookDataArgs, cls).__new__(
+            cls, notebook_path=check.str_param(notebook_path, "notebook_path")
+        )
```

Comment:
slightly dubious about having an object wrapper for a string param - seems unlikely we would want other serdes objects as args here - if ever any other args